command/git_test.go

```diff
@@ -661,7 +663,62 @@ func (s *GitGetProjectSuite) TestCorrectModuleRevision() {
 	err = cmd.Run()
 	s.NoError(err)
 	ref := strings.Trim(out.String(), "\n")
-	s.Equal("b27779f856b211ffaf97cbc124b7082a20ea8bc0", ref) // this revision is defined in the patch, returned by GetTaskPatch
+	s.Equal(correctHash, ref) // this revision is defined in the patch, returned by GetTaskPatch
+	s.NoError(logger.Close())
+	toCheck := `Using revision/ref 'b27779f856b211ffaf97cbc124b7082a20ea8bc0' for module 'sample' (reason: specified in set-module)`
+	foundMsg := false
+	for _, task := range comm.GetMockMessages() {
+		for _, msg := range task {
+			if msg.Message == toCheck {
+				foundMsg = true
+			}
+		}
+	}
+	s.True(foundMsg)
+}
+
+func (s *GitGetProjectSuite) TestCorrectModuleRevision2() {
```

Comment:
Can this be something more descriptive for the future? UsingManifest maybe?

Reply:
Renamed both tests to be more descriptive